kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerImpl.java

```diff
@@ -196,6 +203,31 @@ protected void init(StartupStrategy startupStrategy) {
         eventSupport.fireAfterServerStarted(this);
     }
 
+    /**
+     * we allow some config items to be overwritten by properties
+     * @param currentState
+     */
+    private boolean updateMutableEnvironmentState(KieServerState currentState) {
+        boolean updated = false;
+        for(String key : Arrays.asList(KieServerConstants.CFG_KIE_CONTROLLER_TOKEN, KieServerConstants.CFG_KIE_CONTROLLER_PASSWORD, KieServerConstants.CFG_KIE_CONTROLLER_TOKEN)) {
```

Comment:
Token is repeated and missing user:
```suggestion
        for(String key : Arrays.asList(KieServerConstants.CFG_KIE_CONTROLLER_USER, KieServerConstants.CFG_KIE_CONTROLLER_PASSWORD, KieServerConstants.CFG_KIE_CONTROLLER_TOKEN)) {
```

Reply:
done